helm/squid-proxy/templates/service.yaml

```diff
@@ -3,7 +3,7 @@ kind: Service
 metadata:
   annotations:
   {{- if .Values.baseDomain }}
-    external-dns.alpha.kubernetes.io/hostname: "proxy.{{ .Values.baseDomain }}"
+    external-dns.alpha.kubernetes.io/hostname: "proxy.{{ .Values.clusterID }}.{{ .Values.baseDomain }}"
```

Comment:
where is the value of `clusterID` injected into this chart? I cannot find it in any of other other places

Reply:
well, `baseDomain` is not declared as well.

Reply:
I think the best approach would be fix the value here https://github.com/giantswarm/giantswarm-management-clusters/blob/main/management-clusters/goat/outgoing-proxy-stack-app.yaml#L13

as if you change this chart you would need to change how this app is deployed during mc-bootstrap which will most probably won't have the chart values as on fully running and established MC

Reply:
confirmed it works without the need to declare it